pkg/constants/constants.go

```diff
@@ -171,7 +172,6 @@ func GetGrubModules() []string {
 func GetConfigScanDirs() []string {
 	return []string{
 		"/oem",
-		"/system/oem",
```

Comment:
but we need this somewhere else no? As this is used by the runstage stuff IIRC to read the stages from the whole system

Reply:
Hm you are right. Should we just rely on headers missing in our overlay files or should I remove the directory here: https://github.com/kairos-io/kairos-agent/blob/a8e855b9337fe92688148a6aac2281c3acc2397d/main.go#L330 ?

I mean, the whole idea is that `kairos-agent config` doesn't print values from those files, but that's already the case if we skip because of the missing header. We had one of them with a header though which makes me think that this is error prone and maybe we should skip the dir too?

Reply:
I would drop that and have a separate function that get the YipConfigsDirs that includes all of the above.

Reply:
and use that in RunStage I guess?

Reply:
that's another kairos-sdk release. By the way, these are all irrelevant to this PR. Should we maybe do it separately? This is about bumpind the sdk to show the sources.

Reply:
I reverted the change so that we do it in another PR